pkg/kubelet/network/plugins.go

```diff
@@ -38,6 +38,13 @@ type NetworkPlugin interface {
 	// before any other methods are called.
 	Init(host Host) error
 
+	// Tells the plugin to start any long-running operations; the plugin
+	// should not block this function
+	Run() error
```

Comment:
Init() gets called when the plugin is initially loaded.  Run() is called when more of kubelet is set up, so the plugin can start long running operations.

The cbr0 plugin doesn't care much, so I'll just make it do everything from Run() for now.